rhconsulting_tags.rake

```diff
@@ -28,7 +28,7 @@ class TagImportExport
     if file_type == 'file'
       File.write(filename, Classification.export_to_yaml)
     elsif file_type == 'directory'
-      Classification.find_all_by_parent_id("0").each do |category|
+      Classification.where(parent_id => "0").each do |category|
```

Comment:
I think a : is missing in front of parent_id.  It should be Classification.where(**:parent_id** => "0").each do |category|